### PR TITLE
Update Phylogenetics.md

### PR DESCRIPTION
**Pull request by thijsjanzen**
- add reference to DDD package as package that can simulate
- add reference to secsse package as package that can do trait based simulations
- add section on phylogenetic summary statistics + associated packages

### CODE REVIEW

File: Phylogenetics.md [hunk 1]
```diff
@@ -121,6 +133,7 @@ Packages within the task view fall within one or more of the following task cate
 - `r pkg("TreeSim")` can be used to simulate trees using constant-rate birth-death with various constraints.
 - `r pkg("phytools")` can simulate birth-death trees with various constraints, in both continuous and discrete time.
 - `r pkg("geiger")` can be used to simulate trees under a birth-death process.
+- `r pkg("DDD")` can used to simulate trees under a diversity-dependent diversification process.
```
Comment (willgearty) [inline Phylogenetics.md:136]:
```suggestion
- `r pkg("DDD")` can be used to simulate trees under a diversity-dependent diversification process.
```

File: Phylogenetics.md [hunk 2]
```diff
@@ -92,6 +92,18 @@ Packages within the task view fall within one or more of the following task cate
 - `r pkg("ape")` can compute tree-tree distances and also create a plot showing two trees with links between associated tips.
 - `r pkg("dendextend")` can evaluate multiple measures comparing dendrograms.
 
+### Phylogenetic summary statistics
+
+- `r pkg("treestats")` is a wide collection of tree statistics, optimized for fast calculation.
```
Comment (willgearty) [inline Phylogenetics.md:97]:
```suggestion
- `r pkg("treestats")` can be used to calculate a wide collection of tree statistics, optimized for fast calculation.
```

File: Phylogenetics.md [hunk 3]
```diff
@@ -92,6 +92,18 @@ Packages within the task view fall within one or more of the following task cate
...
+- `r pkg("nLTT")` is specialized in calculating and visualising the nLTT statistic.
+- `r pkg("castor")` contains fast calculation of the Gamma, Colless and Sackin statistic.
+- `r pkg("phyloTop")` is a collection of pattern based summary statistics (e.g. ladders, stairs, cherries, pitchforks).
```
Comment (willgearty) [inline Phylogenetics.md:100]:
```suggestion
- `r pkg("phyloTop")` can be used to calculate a collection of pattern based summary statistics (e.g. ladders, stairs, cherries, pitchforks).
```

File: Phylogenetics.md [hunk 4]
```diff
@@ -92,6 +92,18 @@ Packages within the task view fall within one or more of the following task cate
...
+- `r pkg("treebalance")` is a collection of summary statistics focusing on measuring (im)balance.
```
Comment (willgearty) [inline Phylogenetics.md:101]:
```suggestion
- `r pkg("treebalance")` can be used to calculate a collection of summary statistics focusing on measuring (im)balance.
```

**Review by willgearty — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a couple suggestions to the wording.

File: Phylogenetics.md [hunk 5]
```diff
@@ -92,6 +92,18 @@ Packages within the task view fall within one or more of the following task cate
...
+- `r pkg("RPANDA")` can compute Laplacian spectrum associated statistics.
+- `r pkg("picante")` can compute community level summary statistics, such as mpd, mntd and psv.
+- `r github("Leonardini/treeCentrality")` can compute several statistics inspired from network science.
+- `r pkg("apTreeshape")` can compute Aldous' Beta, Colless and Sackin.
```
Comment (willgearty) [inline Phylogenetics.md:105]: apTreeshape is no longer being maintained (and has been removed from CRAN)
```suggestion
```

**Comment by thijsjanzen** [comment]
> Thanks for the PR! Just a couple suggestions to the wording.

Thanks for the improvements!

**Review by willgearty — APPROVED** [review]
Looks good, thanks again!

**Pull request merged**